### PR TITLE
Updated italian lang

### PR DESCRIPTION
**Pull request by itajackass**
added missing strings

**Merge request checklist**

- [ ] I read the [guidelines for contributing](https://github.com/mistic100/jQuery-QueryBuilder/blob/master/.github/CONTRIBUTING.md)
- [ ] I created my branch from `dev` and I am issuing the PR to `dev`
- [ ] I didn't pushed the `dist` directory
- [ ] Unit tests are OK
- [ ] If it's a new feature, I added the necessary unit tests
- [ ] If it's a new language, I filled the `__locale` and `__author` fields

### CODE REVIEW

File: src/i18n/it.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "__locale": "Italian (it)",
-  "__author": "davegraziosi",
+  "__author": "Giuseppe Lodi Rizzini",
```
Comment (mistic100) [inline src/i18n/it.json]: By courtesu please to not erase the name of previous contributors. You can put both names.

**Comment by itajackass** [comment]
Ok thanks

**Comment by mistic100** [comment]
@itajackass ok you resolved the conversation but did not changed anything

**Comment by itajackass** [comment]
> 
> 
> @itajackass ok you resolved the conversation but did not changed anything

Sorry i'm not very good with github. I think now I have edited correctly.

**Pull request merged**